src/deprecated/deprecated.js

```diff
@@ -662,6 +663,21 @@ Object.defineProperty(StandardMaterial.prototype, 'useGammaTonemap', {
     }
 });
 
+Object.defineProperty(StandardMaterial.prototype, 'anisotropy', {
```

Comment:
please add Debug.deprecated to both the getter and setter, to warn user there is a newer API to use.

Reply:
Fixed at https://github.com/playcanvas/engine/pull/7668/commits/29f56d6a68c1fe355e96ddb27745d15f08d875c1